src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -19,6 +19,9 @@ type ThorTradeQuoteSpecificMetadata = {
   isStreaming: boolean
   memo: string
   recommendedMinimumCryptoBaseUnit: string
+  longtailData?: {
+    longtailToLExpectedAmountOut?: bigint
```

Comment:
```suggestion
    longtailToL1ExpectedAmountOut?: bigint
```

Reply:
Good eyes. Fixed in https://github.com/shapeshift/web/pull/6052/commits/436e8b7537e640938e5900bcec78221090702f70.